### ISSUE TITLE
Invalid build output for `strategy: 'unbundle'`

### ISSUE THREAD

**Issue by nichita-pasecinic** [sanyuan0704/vite-plugin-chunk-split#7]
On `"vite": "^2.9.7"` and with `unbundle` strategy the build will not run in preview mode (with `npm run preview`)
Instead it generates a whole bunch of 404 not found errors. I assume it is because of the wrong path in `index.html`

![image](https://user-images.githubusercontent.com/98735377/182309501-fbbf836b-a778-4ba1-9ba3-9b0396ad618a.png)

example of `index.html` script paths (`unbundle` strategy)

```html
<script type="module" crossorigin src="/assets/vite\\modulepreload-polyfill.75a5f89c.js"></script>
<script type="module" crossorigin src="/assets/react\\jsx-runtime.7eec44ec.js"></script>
<script type="module" crossorigin src="/assets/_commonjsHelpers.0592d25c.js"></script>
<script type="module" crossorigin src="/assets/_react\\jsx-runtime_commonjs-proxy.b6872daf.js"></script>
<script type="module" crossorigin src="/assets/vendor.4581de72.js"></script>
<script type="module" crossorigin src="/assets/src\\config\\brand-config.3bbd6094.js"></script>
<script type="module" crossorigin src="/assets/src\\utils\\generate-utility-classnames.56b99e79.js"></script>
<script type="module" crossorigin src="/assets/src\\pages\\welcome\\components\\constants.d4fff9c7.js"></script>
<script type="module" crossorigin src="/assets/src\\utils\\styles.cb43d375.js"></script>
<script type="module" crossorigin src="/assets/src\\themes\\dark.45618e04.js"></script>
<script type="module" crossorigin src="/assets/src\\themes\\light.d28e6639.js"></script>
```

example from `index.html` script path (no strategy)

```html
<script type="module" crossorigin src="/assets/index.6cbdf144.js"></script>
<link rel="modulepreload" href="/assets/vendor.cac78811.js">
<link rel="stylesheet" href="/assets/index.173328f5.css">
```

**Comment by sanyuan0704** [comment]
Has any reproduction repo?

**Comment by sanyuan0704** [comment]
I found double slash.Do you use windows system?

**Comment by sanyuan0704** [comment]
I have published 0.2.6 to compat windows system @nichita-pasecinic
